index.test.js

```diff
@@ -149,6 +153,57 @@ describe('Deploy to ECS', () => {
                 }
             };
         });
+
+        mockRunTasks.mockImplementation(() => {
+            return {
+                promise() {
```

Comment:
The mock needs to be updated as the action is now using JS SDK v3

Refer: https://github.com/aws-actions/amazon-ecs-deploy-task-definition/pull/529